core/repository/transaction.ts

```diff
@@ -17,13 +18,60 @@ interface IDBTransactionSave {
 
 export interface ITransactionRepository<T extends Object> extends ITransactionRepositoryShared<T> {
     list(): Array<Transaction<T>>;
+
+    getTransactionsForBlocksByIds(ids: Array<string>):
+        Promise<Response<{ [blockId: string]:  Array<Transaction<object>> }>>;
 }
 
 class TransactionRepo implements ITransactionRepository<object> {
     one(): Promise<Transaction<object>> {
         return null;
     }
 
+    async  getTransactionsForBlocksByIds(ids: Array<string>):
+        Promise<Response<{ [blockId: string]:  Array<Transaction<object>> }>> {
+        let result: { [blockId: string]:  Array<Transaction<object>> } = {};
+        try {
+            const rawTransactions: Array<object> = await db.manyOrNone(queries.getTransactionsForBlocksByIds, [ids]);
+            if (!result) {
+                return new Response({ errors: ['No blocks found']});
```

Comment:
No transactions found. But block may not have transactions and we must return an empty array